specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,731 @@
+                        "required": false
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/TokenResponse"
+                        }
+                    },
+                    "400": {
+                        "description": "Invalid request",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "404": {
+                        "description": "Not found",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "405": {
+                        "description": "Method not allowed",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "429": {
+                        "description": "Too many requests",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "500": {
+                        "description": "Server error",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get Identity Token for the VM": {
+                        "$ref": "./examples/getidentitytoken.json"
+                    }
+                }
+            }
+        },
+        "/identity/info": {
+            "get": {
+                "tags": [
+                    "Get metadata information"
+                ],
+                "summary": "Get information about AAD Metadata",
+                "description": "",
+                "operationId": "getInfo",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "Metadata",
+                        "in": "header",
+                        "type": "string",
+                        "enum": [
+                            "true"
+                        ],
+                        "description": "This must be set to 'true'.",
+                        "required": true
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "type": "string",
+                        "enum": [
+                            "2018-02-01",
+                            "2018-04-02",
+                            "2018-10-01"
```

Comment:
Usually the api-version for all the apis in a given document remains constant. Why does this api-version parameter have an enum whereas others do not?

Reply:
Added enum to other instances of the parameter.